Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py

```diff
@@ -652,7 +675,19 @@ def main():
             'sql-command': sql_query_execute
         }
         if command in commands:
-            return_outputs(*commands[command](client, demisto.args(), command))
+            command_results: list[CommandResults] = []
+            human_readable, outputs, raw = commands[command](client, demisto.args(), command)
+            result = CommandResults(readable_output=human_readable,
+                                    entry_type=entryTypes["note"],
+                                    outputs=outputs,
+                                    raw_response=raw,
+                                    ignore_auto_extract=False,
+                                    content_format=formats["text"] if isinstance(raw, STRING_TYPES) else formats['json']
+                                    )
+
+            command_results.append(result)
+            return_results(command_results)
+
```

Comment:
Do you need this for support with the execution metrics? if not, maybe it's worth not to change it to not break something mistakenly

Reply:
I reverted this block back to the deprecated command.